server/controllers/users.js

```diff
@@ -96,3 +96,24 @@ export function getAllUsers(req, res, next) {
       console.log('could not retrieve users', err);
     });
 }
+
+export function manageUserById(req, res, next) {
+  const params = {
+    id: req.params.id,
+    isAdmin: req.body.is_admin,
+    isActive: req.body.is_active,
+    isBanned: req.body.is_banned
+  };
```

Comment:
We should reject the request if is_active and is_banned are both set.

Reply:
Further, I think a ban should automatically deactivate the user as well.

Reply:
We would also need tests around that.

Reply:
I'm on it.